libcxx/test/std/utilities/tuple/tuple.general/ignore.include.compile.pass.cpp

```diff
@@ -0,0 +1,21 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03
+
+// <tuple>
```

Comment:
Can you add a comment along the lines of
```
std::ignore should be provided by the headers <tuple> and <utility>.
This tests validates its presence in <tuple>.
```

Reply:
The paper also "recommends" to mention the same in `<tuple>`. Do you have any recommendations what and where to write?

> YES, make ignore available through <utility>, and not repeat the specification, but follow the example of ranges accessor functions by saying in [tuple] somewhere that ignore is available also from header <utility>.

Reply:
I'm not entirely sure what you mean with "The paper also "recommends" to mention the same in `<tuple>`"

The wording section contains 
```
In [tuple.general](https://eel.is/c++draft/tuple.general) append the following paragraph

<ins>In addition to being available via inclusion of the <tuple> header, ignore ([tuple.syn]) is available when <utility> ([utility]) is included.</ins>
```
This is not a recommendation but a requirement.

Can you explain what your question is? Feel free to do it on Discord if you prefer that.